packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -352,10 +352,12 @@ function registerOrgPickerCommands(orgList: OrgList): vscode.Disposable {
 }
 
 async function setupOrgBrowser(
```

Comment:
Can this setup function be defined in the orgBrowser module and imported instead? Might be good to reduce crowding this file when we can.

Reply:
I was thinking about that initially too but ended up deciding against it for now. If we want to move these helper setup methods over to their respective files, I can definitely do that in another pr.